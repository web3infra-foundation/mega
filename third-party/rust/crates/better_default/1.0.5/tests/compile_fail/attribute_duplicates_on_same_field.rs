#![allow(dead_code)]

use better_default::Default;

#[derive(Default)]
enum Enum2 {
    #[default]
    #[default]
    Variant {
        first: u32,
        second: String,
    },

    Variant2,

    Variant3,
}

#[derive(Default)]
struct Struct {
    #[default(0)]
    #[default(1)]
    field: u32
}

fn main() {}