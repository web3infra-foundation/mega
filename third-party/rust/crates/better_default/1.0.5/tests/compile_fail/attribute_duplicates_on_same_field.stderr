error: this attribute is already declared.
 --> tests/compile_fail/attribute_duplicates_on_same_field.rs:8:7
  |
8 |     #[default]
  |       ^^^^^^^

error: this attribute is already declared.
  --> tests/compile_fail/attribute_duplicates_on_same_field.rs:22:7
   |
22 |     #[default(1)]
   |       ^^^^^^^
